app/routes/admin/email/components/EmailUserEditor.tsx

```diff
@@ -72,17 +72,17 @@ const EmailUserEditor = ({
         placeholder="abakus"
         suffix="@abakus.no"
         name="internalEmail"
-        label="Gsuite Epost"
+        label="G Suite e-post"
```

Comment:
Google is calling this product Google Workspace now, personally I think a hyphenated G Suite-e-post could work, but `Google Workspace-e-post` might be best. I'd even say `Google Workspace e-post` looks better in this case.

Reply:
Yea I saw that, but since _we_ still refer to it as G Suite I thought it was best to not "change" here. I'll note it for later though.